arduino/arduino-ide
# chore(deps): update `electron`+`electron-builder`

### Motivation


This PR updates the version of the `electron` and `electron-builder` dependencies. I created this PR as the first attempt to fix https://github.com/eclipse-theia/theia/issues/12751.

### Change description


This PR updates the dependencies to `electron@25.5.0` and `electron-builder@24.6.3`.

### Other information


Related error messages and issues:

```
Gtk-Message: 15:40:49.964: Failed to load module "xapp-gtk3-module"
```

```
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
```

(when started with `--disable-gpu`):
```
[55758:0725/145858.450066:ERROR:gpu_init.cc(453)] Passthrough is not supported, GL is swiftshader, ANGLE is 
```

To verify:
 - CI works,
 - after downloading and starting the tester build, verify the `electron` version:
    - Open the _Dev Tools_ in IDE2 (<kbd>Ctrl/⌘</kbd>+<kbd>Alt</kbd>+<kbd>I</kbd>),
    - Open _Console_ (with <kbd>Esc</kbd>) if not opened,
    - Type: `navigator.userAgent`, and press <kbd>Enter</kbd>.
    - It must be `Electron/25.5.0`. The rest could be different per OS:

<img width="357" alt="Screenshot 2023-08-14 at 14 20 14" src="https://github.com/arduino/arduino-ide/assets/1405703/ff04973c-97ae-49d7-b0bc-21237bd3013c">


### Reviewer checklist
